### DISCUSSION TITLE
Draggable Table Columns

### DISCUSSION THREAD

**Discussion by MACscr** [filamentphp/filament#3386] [Ideas]
Its great that we can now toggle table columns, but would also be useful when you are dealing with a lot of columns, is to be able drag the columns left to right to change their order. Almost like you would in Excel, etc.

**Comment by MarJose123** [comment]
Hoping we can have this feature maybe before v3 😃

**Comment by danharrin** [comment]
Reorderable table columns is now a feature in v4
